File: Lv1/가운데_글자_가져오기/Solution.js
function solution(s) {
    let answer = '';
    const sLength = s.length;
    const sMid = sLength /2;
    if(sLength % 2 == 1 ){
        answer = s.substring(sMid, sMid + 1);

    }else{
        answer = s.substring(sMid -1 , sMid + 1);
    }
    return answer;
}

File: Lv1/가운데_글자_가져오기/Solution.java
package Lv1.가운데_글자_가져오기;

class Solution {
    public String solution(String s) {
        String answer = "";
        int stringLength = s.length();
        int stringMid = stringLength / 2;
        if (stringLength % 2 == 1) {
            answer = s.substring(stringMid, stringMid + 1 );
        }else{
            answer = s.substring(stringMid -1 , stringMid + 1 );
        }
        return answer;
    }
}